### DISCUSSION TITLE
Where is diff tool specified?

### DISCUSSION THREAD

**Discussion by ollypsilon** [joyfullservice/msaccess-vcs-addin#382] [Q&A] [answered]
I tried the "Show Difference" option but after fixing a wrong code part I still didn't get any results displayed. So I investigated further.
There is code to read and set diff tool to be used.
https://github.com/joyfullservice/msaccess-vcs-integration/blob/6fedf269e9593b4ad043ab35b43b8b819ef71c32/Version%20Control.accda.src/modules/clsViewDiff.cls#L114

https://github.com/joyfullservice/msaccess-vcs-integration/blob/6fedf269e9593b4ad043ab35b43b8b819ef71c32/Version%20Control.accda.src/modules/clsViewDiff.cls#L128

For me it wasn't set so I set it by `SaveSetting "MSAccessVCS","Options", "Diff Tool", "WinMerge"` in direct window and WinMerge was opened showing diff between both files.
I wonder if this should be an option in VCS Options screen?
Or how will it be set?

**Comment by joyfullservice** [comment, marked as answer]
Yes, this part wasn't fully finished yet.  😄 Good job figuring out how to set it manually. We will probably add it to the options screen as a dropdown showing the installed tools.

**Reply by joyfullservice** [reply]
I have now exposed this setting on the Options screen...

![image](https://user-images.githubusercontent.com/7254992/220777910-d906bc5e-6e74-4885-891a-79939aa19d28.png)
